app/_layout.tsx

```
@@ -14,6 +14,8 @@ import { PortalProvider } from '@gorhom/portal';
 // Prevent the splash screen from auto-hiding before asset loading is complete.
 SplashScreen.preventAutoHideAsync();
 
+LogBox.ignoreLogs([new RegExp('StatusBar backgroundColor is not supported with edge-to-edge enabled. Render a view under the status bar to change its background.')]);
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)